[2024-05-24T00:20:57.984+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-24T00:20:57.995+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.extract_Sea_Level_Data manual__2024-05-24T00:20:57.409182+00:00 [queued]>
[2024-05-24T00:20:57.999+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.extract_Sea_Level_Data manual__2024-05-24T00:20:57.409182+00:00 [queued]>
[2024-05-24T00:20:57.999+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-24T00:20:58.007+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_Sea_Level_Data> on 2024-05-24 00:20:57.409182+00:00
[2024-05-24T00:20:58.011+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=190) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-24T00:20:58.012+0000] {standard_task_runner.py:63} INFO - Started process 200 to run task
[2024-05-24T00:20:58.012+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'extract_Sea_Level_Data', 'manual__2024-05-24T00:20:57.409182+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/***_dag.py', '--cfg-path', '/tmp/tmpetdrtrre']
[2024-05-24T00:20:58.013+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask extract_Sea_Level_Data
[2024-05-24T00:20:58.036+0000] {task_command.py:426} INFO - Running <TaskInstance: data_pipeline.extract_Sea_Level_Data manual__2024-05-24T00:20:57.409182+00:00 [running]> on host 7f18484bb9a3
[2024-05-24T00:20:58.077+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='extract_Sea_Level_Data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T00:20:57.409182+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-24T00:20:57.409182+00:00'
[2024-05-24T00:20:58.078+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-24T00:20:59.434+0000] {log.py:232} WARNING - [2024-05-24 00:20:59,434] [INFO] - Data extracted and saved to: ./data/raw/
[2024-05-24T00:20:59.434+0000] {base_logger.py:19} INFO - Data extracted and saved to: ./data/raw/
[2024-05-24T00:20:59.434+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-24T00:20:59.434+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-24T00:20:59.438+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=extract_Sea_Level_Data, run_id=manual__2024-05-24T00:20:57.409182+00:00, execution_date=20240524T002057, start_date=20240524T002057, end_date=20240524T002059
[2024-05-24T00:20:59.469+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-24T00:20:59.487+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-24T00:20:59.489+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
